src/components/Layout/Footer/FooterLayout.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC } from "react";
+
+//import "@mozilla-protocol/core/protocol/css/protocol.css";
+interface Link {
+  label: string;
+  url: string;
+}
+
+interface Header {
+  links: Link[];
+  headerName: string;
+}
+
+export interface FooterLayoutProps {
+  /**The links that go in the footer
+   * example link object: {
+   *    about: <string>
+   *    aboutLinkDestination: <string>
+   *    headers: [
+   *            {
+   *              headerName: "City of Seattle",
+   *              links: Link[]
+   *          }
+   *    ]
+   * }
+   */
+  headers: Header[];
+  about: string;
+  aboutLinkDestination: string;
+}
+
+function renderHeader(header: Header) {
+  return (
+    <section className="mzp-c-footer-section">
+      <h2 className="mzp-c-footer-heading">{header.headerName}</h2>
+      <ul style={{ fontSize: "12px" }}>
+        {header.links.map((link) => {
+          return (
+            <li key={link.label}>
+              <a href={link.url}>{link.label}</a>
+            </li>
+          );
+        })}
+      </ul>
+    </section>
+  );
+}
+
+const FooterLayout: FC<FooterLayoutProps> = ({
+  headers,
+  about,
+  aboutLinkDestination,
+}: FooterLayoutProps) => {
+  return (
+    <footer className="mzp-c-footer">
+      <div className="mzp-l-content">
+        <section className="mzp-c-footer-section">
+          <h1 className="mzp-c-footer-heading" style={{ fontSize: "1.65rem" }}>
+            Council Data Project
+          </h1>
+          <p>{about}</p>
+          <form action={aboutLinkDestination}>
+            <input
+              className="mzp-c-button mzp-t-product"
+              type="submit"
+              value="About the project"
+              style={{
+                borderRadius: ".25rem",
+                border: "none",
+                height: "2.25rem",
+                backgroundColor: "#307D7E",
+                color: "white",
+                fontWeight: "bold",
+              }}
+            />
+          </form>
+        </section>
```

Comment:
Yeah, I agree. I like the idea but since people will very likely see all of this information elsewhere I don't it necessarily needs to be repeated in the footer.